src/containers/ControlledSwitch/index.js

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+import './switch.scss';
+
+const Switch = ({ isOn, toggle }) => (
+	<div className="switch-container">
+		<div className={isOn ? "on" : "off"} onClick={toggle}>
```

Comment:
@titkov49 Dima, I miss the option to disable the switch. Also there is no `title` to display some info about the switch. And last but not least -  `isOn` variable should be a `boolean` so we can set the `value` to `true` or `false`, otherwise the `isOn` name does not make a sense, if the string values can be `on` or `off`.